includes/cli/backfillers/class-reader-registered.php

```diff
@@ -49,6 +55,8 @@ public function get_events() {
 			]
 		);
 
+		WP_CLI::line( sprintf( 'Found %s users eligible for sync.', count( $users ) ) );
```

Comment:
We need to account for single user (`Found 1 user eligible for sync`)

Reply:
Fixed in 121cc686236bac0458e5a8e811fa2380fca29171